install.sh

```diff
@@ -25,26 +25,27 @@ fi
 # Reject crostini.
 if [[ -d /opt/google/cros-containers && "${CREW_FORCE_INSTALL}" != '1' ]]; then
   echo_error "Crostini containers are not supported by Chromebrew :/"
-  echo_info "Run 'CREW_FORCE_INSTALL=1 exec bash --init-file <(curl -Ls git.io/vddgY)' to perform install anyway"
+  echo_info "Run 'CREW_FORCE_INSTALL=1 bash <(curl -L git.io/vddgY) && . ~/.bashrc' to perform install anyway"
```

Comment:
Simplify further with?
```
... curl -Ls git.io/vddgY | bash ...
```

Reply:
Using pipes will break things like `select` and `read`, as stdin will be redirected to `curl` and no longer connected to the terminal

https://github.com/chromebrew/chromebrew/blob/f78af8f68315bfd6ae2f3017826064e82272f10d/install.sh#L57-L59

Reply:
Gotcha, good to know.